engine/class_modules/sc_mage.cpp

```diff
@@ -8453,6 +8487,8 @@ void mage_t::create_buffs()
                                       ->set_default_value_from_effect( 1 )
                                       ->modify_default_value( talents.aether_fragment->effectN( 1 ).percent() )
                                       ->set_chance( talents.intuition.ok() );
+  buffs.intuition_tracker         = make_buff( this, "intuition_tracker", find_spell( 384455 ) ) //uses arcane harmony's id, scuffed but works well enough (i think, but it's a HUGE hack-job), im not sure how to set up an imaginary buff without literally adding it into spelldata. previously did use intuition's id, had to change its duration as to not have it expire, but it's also way neater within the html report -- i dunno
```

Comment:
There are ways of creating "fake" buffs like this, you've already discovered that you can manually set stacks. You can do the same thing with duration etc. But as I mentioned earlier, this would work much better as a field in the `state` object.